elastic4s-tcp/src/main/scala/com/sksamuel/elastic4s/searches/queries/QueryStringBuilderFn.scala

```diff
@@ -15,7 +15,7 @@ object QueryStringBuilderFn {
     query.defaultField.foreach(builder.defaultField)
     query.enablePositionIncrements.foreach(builder.enablePositionIncrements)
     query.fields.foreach {
-      case (name, -1) => builder.field(name)
+      case (name, 1D) => builder.field(name)
       case (name, boost) => builder.field(name, boost.toFloat)
```

Comment:
Note, this seems to be a subtle bug by itself since it maches on an Int and never reaches the first case.